src/util/piped_process.h

```diff
@@ -0,0 +1,39 @@
+#ifndef CPROVER_UTIL_PIPED_PROCESS_H
+#define CPROVER_UTIL_PIPED_PROCESS_H
+
+// TODO: Revisit buffer size and storage location.
+// Note that this may impact number of characters correctly communicated
+// by the pipe, to check/fix.
+#define BUFSIZE 2048
```

Comment:
:grey_question: Is this just an implementation detail that should be in the `.cpp` file? I am thinking about the increased compilation time for anyone tuning this parameter later.